app/helpers/textual_mixins/textual_power_state.rb

```diff
@@ -1,9 +1,22 @@
 module TextualMixins::TextualPowerState
-  def textual_power_state
-    state = @record.current_state.downcase
-    state = "unknown" if state.blank?
-    h = {:label => _("Power State"), :value => state}
-    h[:image] = "svg/currentstate-#{@record.template? ? 'template' : state}.svg"
-    h
+  VALID_POWER_STATE = %w(
```

Comment:
This looks suspiciously similar to `QuadiconHelper#MACHINE_STATE_QUADRANT`

Should we try to unify those two as well?

Reply:
Similar, but not the same. Yes, we could and should unify that. But I don't have an authoritative set of values for this. That would have to come from the backend. Or do we know what are the valid valies? @skateman ?

I see two options: 
  * leave this as it is now. Possibly do some unification in the user, after we somehow figure what are the valid values.
  * use @skateman 's list from `QuadionHelper` ignoring the couple of extra values (`powering*`).

I don't have a strong opinion here.

Reply:
Using this for now, eventually we should be using fonticons instead of svgs here as well.